README.md

```diff
@@ -16,6 +16,9 @@ An interactive terminal interface for [Zulip](https://zulipchat.com).
 - OSX
 - WSL (On Windows)
 
+## Image of Zulip-Terminal
+![image](https://user-images.githubusercontent.com/56690786/82187341-e4c39600-9909-11ea-9037-165f24d2691e.png)
```

Comment:
Thanks for adding this here too :+1:

I think we likely want to place this above the `Supported Platforms` to get this more visibility, and we don't really need a heading for it?

Reply:
Sure